IntegrationTests/GoldMaster/Calabash.podspec.json.goldmaster

```diff
@@ -84,7 +85,7 @@ objc_library(
     }
     ),
   copts = [
-    ""$(PODS_ROOT)/Calabash/calabash.framework/calabash"",
+    ""Vendor/Calabash/calabash.framework/calabash"",
```

Comment:
Extra quotes?

Reply:
Looks like this entire example is busted 🔥